apps/preview/next/pages/showcases/NavbarTop/NavbarTop.tsx

```diff
@@ -27,39 +39,82 @@ export default function TopNav() {
-            </picture>
-          </a>
-          <SfButton className="block mr-auto" type="button" slotSuffix={<SfIconExpandMore />} variant="tertiary">
-            <span className="hidden md:inline-flex">Browse products</span>
-          </SfButton>
-          <nav className="flex flex-row flex-nowrap">
+    <header className="flex justify-center w-full py-2 px-4 lg:py-5 lg:px-6 bg-white border-b border-neutral-200">
+      <div className="flex flex-wrap lg:flex-nowrap items-center flex-row justify-start h-full max-w-[1536px] w-full">
+        <a
+          href="/"
+          aria-label="SF Homepage"
+          className="inline-block mr-4 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm shrink-0"
+        >
+          <picture>
+            <source srcSet={brandLogo.src} media="(min-width: 768px)" />
+            <img
+              src={brandLogoSign.src}
+              alt="Sf Logo"
+              className="w-8 h-8 md:h-6 md:w-[176px] lg:w-[12.5rem] lg:h-[1.75rem]"
+            />
+          </picture>
+        </a>
+        <SfButton
+          aria-label="Open categories"
+          className="lg:hidden order-first lg:order-1 mr-4"
+          square
+          variant="tertiary"
+        >
+          <SfIconMenu />
+        </SfButton>
+        <SfButton
+          className="hidden lg:flex lg:mr-4"
+          type="button"
+          variant="tertiary"
+          slotSuffix={<SfIconExpandMore className="hidden lg:block" />}
+        >
+          <span className="hidden lg:flex whitespace-nowrap">Browse products</span>
+        </SfButton>
+        <nav className="flex-1 flex justify-end lg:order-last lg:ml-4">
+          <div className="flex flex-row flex-nowrap">
             {actionItems.map((actionItem) => (
               <SfButton
-                className="mr-2 -ml-0.5 text-primary-700 hover:bg-primary-100 active:bg-primary-200 hover:text-primary-600 active:text-primary-700"
                 key={actionItem.ariaLabel}
+                className="mr-2 -ml-0.5 rounded-md text-primary-700 hover:bg-primary-100 active:bg-primary-200 hover:text-primary-600 active:text-primary-700"
                 aria-label={actionItem.ariaLabel}
                 variant="tertiary"
-                slotPrefix={actionItem.icon}
                 square
+                slotPrefix={actionItem.icon}
               >
-                {actionItem.role === 'login' && <p className="hidden md:inline-flex">{actionItem.label}</p>}
+                {actionItem.role === 'login' && (
+                  <p className="hidden xl:inline-flex whitespace-nowrap">{actionItem.label}</p>
+                )}
               </SfButton>
             ))}
-          </nav>
-        </div>
-      </header>
-    </div>
+          </div>
+        </nav>
+        <form role="search" className="flex flex-[100%] lg:order-3 mt-2 lg:mt-0 pb-2 lg:pb-0">
```

Comment:
Could you add `onSubmit` method that preventDefault and shows an alert to inform the user that something happened?

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2638/commits/cda4e3ff11f0b72ce8b5388ac02a5c7a88eb169b